### DISCUSSION TITLE
TypeError: Math is undefined / Uncaught TypeError: Math is undefined

### DISCUSSION THREAD

**Discussion by sunilkogo** [vercel/next.js#59777] [Help]
### Summary

I am facing this issue after deploying it on production
My project version is => 14.0.3 and node version is 20.6.1
<img width="1440" alt="Screenshot 2023-12-19 at 11 11 59 PM" src="https://github.com/vercel/next.js/assets/120090764/7c295280-59a6-4d96-962b-c4970b72a707">

**Comment by nicosammito** [comment]
Something overrides Math in your code.

Execute Math in your console. Execute context menu on it and select "Show function definition". This will probably lead you to "Sources" panel and show script where it is being overriden.
